Form/EventListener/ResizeFormListener.php

```diff
@@ -63,8 +63,8 @@ public static function getSubscribedEvents()
     {
         return array(
             FormEvents::PRE_SET_DATA    => 'preSetData',
-            FormEvents::PRE_BIND        => 'preBind',
-            FormEvents::BIND            => 'onBind',
+            FormEvents::PRE_SUBMIT      => 'preSubmit',
```

Comment:
Ok I'll do this.

Note: 
https://github.com/sonata-project/SonataCoreBundle/blob/master/Form/EventListener/FixCheckboxDataListener.php#L53

This BC trick will not works because `preSubmit` is called.

Do you want me to fix it?